pkg/api/pq_helper.go

```diff
@@ -0,0 +1,166 @@
+package api
```

Comment:
This file was introduced from a 3rd party project, so we should include a copyright notice on top of it.
```suggestion
// This file was taken from the pq project and is licensed under the terms of the MIT license: https://github.com/lib/pq
// Copyright (c) 2011-2013, 'pq' Contributors Portions Copyright (C) 2011 Blake Mizerany
package api
```

Reply:
done, thanks